app/src/main/java/org/p2p/wallet/auth/gateway/api/request/UpdateMetadataRpcRequest.kt

```diff
@@ -0,0 +1,27 @@
+package org.p2p.wallet.auth.gateway.api.request
+
+import java.lang.reflect.Type
+import org.p2p.core.rpc.JsonRpc
+import org.p2p.solanaj.utils.crypto.Base64String
+import org.p2p.wallet.auth.gateway.api.response.UpdateMetadataResponse
+import org.p2p.wallet.utils.Base58String
+
+data class UpdateMetadataRpcRequest(
+    @Transient val ethereumAddress: String,
+    @Transient val userPublicKey: Base58String,
+    @Transient val encryptedMetadata: Base64String,
+    @Transient val timestamp: String,
+    @Transient val requestSignature: String,
```

Comment:
what format has signature? base64 / base58?

Reply:
it is json.toString